modules/core/www/idp/logout-iframe.php

```diff
@@ -115,9 +115,9 @@
 
 $globalConfig = \SimpleSAML\Configuration::getInstance();
 if ($type === 'nojs') {
-    $t = new \SimpleSAML\XHTML\Template($globalConfig, 'core:logout-iframe-wrapper.php');
+    $t = new \SimpleSAML\XHTML\Template($globalConfig, 'core:logout-iframe-wrapper.tpl.php');
```

Comment:
By giving the old PHP-templates a .tpl.php extension, they can be easily ignored by Psalm.